front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -0,0 +1,662 @@
+          handleStatusChange={handleStatusChange}
+        />
+
+        <ProjectNameInput
+          titleMultiloc={projectAttrs.title_multiloc}
+          titleError={titleError}
+          apiErrors={apiErrors}
+          handleTitleMultilocOnChange={handleTitleMultilocOnChange}
+        />
+
+        {/* Only show this field when slug is already saved to project (i.e. not when creating a new project, which uses this form as well) */}
+        {!isNilOrError(project) && project.attributes.slug && (
+          <SlugInput
+            slug={slug}
+            resource="project"
+            apiErrors={apiErrors}
+            showSlugErrorMessage={showSlugErrorMessage}
+            handleSlugOnChange={handleSlugOnChange}
+          />
+        )}
+
+        <StyledSectionField>
+          {!project ? (
+            <ProjectTypePicker
+              projectType={projectType}
+              handleProjectTypeOnChange={handleProjectTypeOnChange}
+            />
+          ) : (
+            <>
+              <SubSectionTitle>
+                <FormattedMessage {...messages.projectTypeTitle} />
+              </SubSectionTitle>
+              <ProjectType>
+                {<FormattedMessage {...messages[projectType]} />}
+              </ProjectType>
+            </>
+          )}
+
+          {!project && (
+            <CSSTransition
+              classNames="participationcontext"
+              in={projectType === 'continuous'}
+              timeout={TIMEOUT}
+              mountOnEnter={true}
+              unmountOnExit={true}
+              enter={true}
+              exit={false}
+            >
+              <ParticipationContextWrapper>
+                <ParticipationContext
+                  onSubmit={handleParcticipationContextOnSubmit}
+                  onChange={handleParticipationContextOnChange}
+                  apiErrors={apiErrors}
+                />
+              </ParticipationContextWrapper>
+            </CSSTransition>
+          )}
+        </StyledSectionField>
+
+        {!isNilOrError(project) && projectType === 'continuous' && (
+          <ParticipationContext
+            projectId={project.id}
+            onSubmit={handleParcticipationContextOnSubmit}
+            onChange={handleParticipationContextOnChange}
+            apiErrors={apiErrors}
+          />
+        )}
+
+        <TopicInputs
+          selectedTopicIds={selectedTopicIds}
+          onChange={handleTopicsChange}
+        />
+
+        <GeographicAreaInputs
+          areaType={areaType}
+          areasOptions={areasOptions}
+          areasValues={areasValues}
+          handleAreaTypeChange={handleAreaTypeChange}
+          handleAreaSelectionChange={handleAreaSelectionChange}
+        />
+
+        <Outlet
+          id="app.components.AdminPage.projects.form.additionalInputs.inputs"
+          projectAttrs={projectAttrs}
+          onProjectAttributesDiffChange={handleProjectAttributeDiffOnChange}
+        />
```

Comment:
Not passing authUser via props anymore (don't see the reason) and replaced the onChange with a better typed + more readable version (handleProjectAttributeDiffOnChange).